about.html

```diff
@@ -15,35 +15,79 @@
   <body>
     <header class="main-header">
```

Comment:
Dear @AdnanOlarmmi.
- [x] I have not merged the current branch of my repository to the master branch because of creating the PR. After I merge this branch, that will be updated.